src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    public static void main(String[] arg) {
+
+        CustomFileReader fileReader = new CustomFileReaderImpl();
+        List<String> inputReport = fileReader.read("reportToRead.csv");
```

Comment:
According to the checklist, avoid using hardcoded values for file paths. It's better to define them as constants or pass them as arguments to the main method.